src/main/java/frc/robot/Subsystems/DrivetrainSubsystem.java

```diff
@@ -21,107 +20,102 @@
 
 public class DrivetrainSubsystem extends SubsystemBase{
 
-    ////
-        ADIS16470_IMU IMU;
-        //Pigeon2 IMU;
-        public swerveModule[] modules;
-        SwerveDriveKinematics kinematics;
-        SwerveDriveOdometry odometry;
-        ChassisSpeeds m_chassisSpeeds;
-        double translationMaxAccelerationMetersPerSecondSquared = 25;
-        double rotationMaxAccelerationRadiansPerSecondSquared = 50;
-        SlewRateLimiter translationXLimiter = new SlewRateLimiter(translationMaxAccelerationMetersPerSecondSquared);
-        SlewRateLimiter translationYLimiter = new SlewRateLimiter(translationMaxAccelerationMetersPerSecondSquared);
-        SlewRateLimiter rotationLimiter = new SlewRateLimiter(rotationMaxAccelerationRadiansPerSecondSquared);
-        private final StructArrayPublisher<SwerveModuleState> statePublisher;
+    ADIS16470_IMU IMU;
+    //Pigeon2 IMU;
+    public swerveModule[] modules;
+    SwerveDriveKinematics kinematics;
+    SwerveDriveOdometry odometry;
+    ChassisSpeeds m_chassisSpeeds;
+    double translationMaxAccelerationMetersPerSecondSquared = 25;
+    double rotationMaxAccelerationRadiansPerSecondSquared = 50;
+    SlewRateLimiter translationXLimiter = new SlewRateLimiter(translationMaxAccelerationMetersPerSecondSquared);
+    SlewRateLimiter translationYLimiter = new SlewRateLimiter(translationMaxAccelerationMetersPerSecondSquared);
+    SlewRateLimiter rotationLimiter = new SlewRateLimiter(rotationMaxAccelerationRadiansPerSecondSquared);
+    private final StructArrayPublisher<SwerveModuleState> statePublisher;
+    
     //CONSTRUCTOR//
```

Comment:
How do you feel about this comment? Do you feel that it is helpful to your or others on the team?